RUN3/AliAnalysisTaskAO2Dconverter.cxx

```diff
@@ -253,6 +260,24 @@ void AliAnalysisTaskAO2Dconverter::NotifyRun(){
   if (!triggerContainer) AliFatal("Cannot fetch OADB container for trigger analysis");
   AliOADBTriggerAnalysis* oadbTriggerAnalysis = (AliOADBTriggerAnalysis*) triggerContainer->GetObject(fCurrentRunNumber, "Default");
   fTriggerAnalysis.SetParameters(oadbTriggerAnalysis);
+  //read PHOS trigger bad map
+  AliOADBContainer phosBadmapContainer(Form("phosTriggerBadMap"));
+  phosBadmapContainer.InitFromFile(Form("%s/PHOS/PHOSTrigBadMaps.root", AliAnalysisManager::GetOADBPath()),              
+                                   "phosTriggerBadMap");
+  TObjArray *maps = (TObjArray*)phosBadmapContainer.GetObject(fCurrentRunNumber,"phosTriggerBadMap");
+  if(!maps){
+    AliError(Form("Can not read PHOS Trigger Bad map for run %d. \n",fCurrentRunNumber)) ;    
```

Comment:
fPHOSBadMap will be used below, so this has to be AliFatal, otherwise it will crash anyway later.

However, I wonder will we have such a map also for all MCs (most Run 2 MCs will be converted)? 
If not, the usage of the PHOS bad trigger map should be enabled by a flag in the task.